src/services/ResumeService.ts

```diff
@@ -0,0 +1,58 @@
+import { Response } from 'express';
+import multer from 'multer';
+
+import { AppUser } from '@Entities';
+import { StorageService, StorageServiceImpl } from '@Services';
+
+const fileFilter = (req: Express.Request, file: Express.Multer.File, cb: Function): void => {
+  if (file.mimetype.includes('pdf')) {
+    cb(null, true);
+  } else {
+    console.log('Invalid file type');
+    cb(null, false);
+  }
+};
+
+export const resumeFileUploadOptions = {

```

Comment:
Is there a better spot to place this? It's position here makes it feel like it will be used in the uploadResume function to filter the file but it is actually instead used with the react-controller/multer set up in UserController.